include/tvm/runtime/ndarray.h

```diff
@@ -32,6 +32,10 @@
 
 namespace tvm {
 namespace runtime {
+
+class Allocator;
```

Comment:
remove Allocator(see my other comment about reverse the dependency)

allocator depend on ndarry.h